.gitignore

```diff
@@ -31,6 +31,10 @@ share/python-wheels/
 *.egg
 !tests/integration/source_code/samples/library-egg/*.egg
 MANIFEST
+# TODO: Do we want to keep (generated) requirements.txt in git?
```

Comment:
@nfx : What do you think? Should we keep this in git to be able to upgrade dependencies one by one? Or should we always generate the latest wheelhouse

Reply:
we'll generate wheelhouse on release. we have a `pyproject.toml` to manage dependencies.